encord/user_client.py

```diff
@@ -602,6 +604,40 @@ def __validate_filter(self, properties_filter: Dict) -> Dict:
 
         return ret
 
+    def deidentify_dicom_files(
+        self,
+        dicom_urls: List[str],
+        integration_hash: str,
+    ) -> List[str]:
+        """
+        Deidentify dicom series in external storage.
```

Comment:
You'd want to give an example of what that actually means here. This function should be self documenting.

Reply:
Done